login.html

```diff
@@ -3,9 +3,49 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Login Page</title>
+    <link rel="stylesheet" href="/css/common/auth.css">
+    <link rel="stylesheet" href="/css/common/reset.css">
+    <link rel="stylesheet" href="css/pages/login.css">
+    <title>판다마켓 - 로그인</title>
 </head>
 <body>
-    
+    <main>
+        <a href="/">
+            <img class="logo" src="/assets/logo/logo_1.png" alt="판다마켓 로고">
+        </a>
+        <div class="input_info">
+            <form>
+                <div class="input_group">
+                    이메일<br>
+                    <input type="email" name="email">
+                </div>
```

Comment:
❗️ 수정요청
특정 input이 어떤 값을 받는지 설명하는 text들을 label 이라고 합니다.
이러한 친구들은 일반적으로 마우스로 선택시 연결된 input으로 focus가 이동하는 기능이 있습니다~
이를 위해, 그리고 명확한 연결관계 표시하기 위해 label 태그를 사용해주시는 것이 좋습니다~

```suggestion
                <div class="input_group">
                    <label for="email">이메일</label> // label의 for 속성에 input id 값을 넣어 둘이 연결됨을 명시
                    <input type="email" name="email" id="email">
                </div>
```